nextjs-template/src/services/AppService.ts

```diff
@@ -39,18 +53,23 @@ export default class AppService implements AppServiceInterface {
     this.projectID = this.idBuilder.buildProjectID(projectUID);
   }
 
-  async getEventCount(): Promise<number> { return 0; }
+  async getEventCount(): Promise<number> {
+    return 0;
+  }
```

Comment:
You didn’t change this, but... why does this exist?

Reply:
Not sure - possibly leftover from bulk import code. I've removed